openshift/generate-manifests.sh

```diff
@@ -55,6 +55,8 @@ $KUSTOMIZE build "${TMP_CONFIG}/default" -o "$TMP_KUSTOMIZE_OUTPUT"
 for container_name in "${!IMAGE_MAPPINGS[@]}"; do
   placeholder="${IMAGE_MAPPINGS[$container_name]}"
   $YQ -i "(select(.kind == \"Deployment\")|.spec.template.spec.containers[]|select(.name==\"$container_name\")|.image) = \"$placeholder\"" "$TMP_KUSTOMIZE_OUTPUT"
+  $YQ -i 'select(.kind == "Deployment").metadata.annotations += {"target.workload.openshift.io/management": "{\"effect\": \"PreferredDuringScheduling\"}"}' "$TMP_KUSTOMIZE_OUTPUT"
```

Comment:
I _think_ this is supposed to go in the `spec.template.metadata.annotations`? At least that's where platform operators puts it: https://github.com/openshift/platform-operators/blob/main/manifests/0000_50_cluster-platform-operator-manager_04-rukpak-webhooks.deployment.yaml